### ISSUE TITLE
How to achieve dynamic training?

### ISSUE THREAD

**Issue by swg209** [Klitter/Pyramidal_Person_ReID#5]
@Klitter Hi,  I am trying to achieve the dynamic training, but I meet some problem.  As the paper says,  each iteration choose one sample method  in ID-balanced hard triplet and randomized. So I want to make sure these question:
Is the iteration means epoch?
Is there any achievement of the mutil-loss dynamic  Training code for reference?

**Comment by tomFoxxxx** [comment]
是的。epoch。
第二个问题 没看懂 achievement？ 打错字了吧 老哥

**Comment by furoce** [comment]
他的意思是不是：现有有没有达到论文中多个loss动态训练效果的训练代码？
